packages/cashier-v2/src/AppContent.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Router } from './routes';
+
+const AppContent = () => (
+    <div className='font-sans'>
```

Comment:
If you are trying out the quill-tokens approach, you need to put an ID in your root container, then add `important: #cashier_v2` to the tailwind configuration to increase the specificity for the tailwind classes in cashier-v2
```suggestion
    <div id="cashier_v2' className='font-sans'>
```
Please do test this out first as well first, it worked on my side but I'm not 100% sure it will be on your side

Reply:
hey @adrienne-deriv 
i have applied this fix to cashier-v2 package
thanks for advice🙏🏻